src/scss/objects/_tables.scss

```diff
@@ -1,102 +1,105 @@
- @mixin tables() {
-    /**
-    * Objects > Tables
-    * =================================
-    * Default Table styles
-    *
-    * This is an optional component within Fozzie.
-    * If you'd like to use it in your project you can include it by adding `@include tables();` into your SCSS dependencies file.
-    *
-    * File Index:
-    * - Table specific variables
-    * - Default table styles
-    *
-    * Documentation:
-    * TBC
-    */
+@use '~@justeat/pie-design-tokens/dist/jet' as dt;
+@use '../settings/variables' as v;
 
-    /**
-    * Define associated Table variables
-    */
-    $table-bgColor                : $color-container-default !global; // Default background color used for all tables.
-    $table-bgColor--accent        : $color-grey-10 !global; // Background color used for `.table-striped`.
-    $table-border--color          : $color-grey-10 !global; // Border color for table and cell borders.
-    $table-border--width          : 2px !global; // Border width for table border.
-    $table-innerBorder--color     : $color-grey-40 !global;
-    $table-verticalBorder--color  : $color-grey-50 !global;
+@mixin tables() {
+  /**
+  * Objects > Tables
+  * =================================
+  * Default Table styles
+  *
+  * This is an optional component within Fozzie.
+  * If you'd like to use it in your project you can include it by adding `@include tables();` into your SCSS dependencies file.
+  *
+  * File Index:
+  * - Table specific variables
+  * - Default table styles
+  *
+  * Documentation:
+  * TBC
+  */
 
+  /**
+  * Define associated Table variables
+  */
+  $table-bgColor                : dt.$color-container-default !global; // Default background color used for all tables.
+  $table-bgColor--accent        : dt.$color-grey-10 !global; // Background color used for `.table-striped`.
```

Comment:
Does the class `.table-striped` still exist?

Reply:
I'm not 100%. Would like to avoid changing or modifying code outside of what this PR should be doing if possible. I can  create a ticket to see if anyone uses it or if we're certain now I can delete it altogether now.